### DISCUSSION TITLE
Laser burns MDF great but Pine terrible. SMAX??

### DISCUSSION THREAD

**Discussion by PushingPulse** [arkypita/LaserGRBL#2053] [Q&A] [answered]
Hello,

I've recently bought the FoxAlien Masuter pro with the 40W laser, so I am somewhat new at this. When etching an image in MDF the image shows up great no matter what I set SMAX to when uploading the image (it does get pretty dark at 100%). When trying to etch the same image and size on a simple piece of pine veneered plywood the image doesn't show up at all in lighter areas and is very dark, burnt, and splotchy in darker areas. PWM is enabled, speed and resolution settings are fine. My question is what do I set SMAX to? On FoxAliens website the lasers has an electric output of 40W and laser output of 10W. In the laserGRBL configuration guide it says the following about the $30 line,

"I suggest putting 255 which is the maximum number of nuances that the hardware can physically do. Another value
that can have a meaning is 1000 (it can be understood as 100.0%) or the value in milliwatts of your laser, for example
4500" 

So do I set the $30 line to 255 and when uploading an image set S-max to 255 as well for full range of PWM? or do I set $30 to 10,000 or 40,000 and set SMAX to the same when uploading an image? Also which "power in milliwatts" do I use for this? the 10W or 40W value?

**Comment by arkypita** [comment, marked as answer]
Yes, you understand right the usage of $30 and smax.
$30 mean "what number you will use to indicate the maximum power" and the smax/smin mean what S number lasergrbl produce as output in correspondence of black and white pixels of the image.

What you don't consider is that MDF does not react linearly to laser power. It doesn't burn until a certain power is not reached and than turn black in a very small range of power level.

So to obtain a grayscale you should use a smin very close to the svalue where is start show visible engraving and smax as much close as possible to where you cannot distinguish blacks.

Or use dithering

**Reply by PushingPulse** [reply]
Got it. thank you
